.github/ISSUE_TEMPLATE/z_release_major.md

```diff
@@ -52,7 +52,7 @@ Keep an eye on the forum for the release announcement in the next couple of week
     - An SRE to review and confirm the documentation on upgrade instructions and breaking changes is sufficient
 - [ ] Until release testing passes, make sure regressions are fixed in `master`, cherry-pick them into the release branch, and release another beta.
 - [ ] Create a release in GitHub from the release branch so it shows up under the [Releases tab](https://github.com/medic/cht-core/releases) with the naming convention `<major>.<minor>.<patch>`. This will create the git tag automatically. Link to the release notes in the description of the release.
-- [ ] Confirm the release build completes successfully and the new release is available on the [market](https://staging.dev.medicmobile.org/builds/releases). Make sure that the document has new entry with `id: medic:medic:<major>.<minor>.<patch>`
+- [ ] Confirm the release build completes successfully and the new release is available on the [market](https://staging.dev.medicmobile.org/builds_4/releases). Make sure that the document has new entry with `id: medic:medic:<major>.<minor>.<patch>`
```

Comment:
The result of cherry-picking the change into the branch is that the diff shows up in the changed files in GitHub. This can be overwhelming if the cherry picked changes cover much volume. 
To avoid this, there are two options: 
1. Merge the target branch in yours.
2. Rebase your branch over the target branch

In my experience, merging is quicker and less error prone, because you only need to resolve conflicts once.

Reply:
Ok, I understand, thanks for the remark  and the advice.